diff-containers/test/Test/Util.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+
+module Test.Util (OftenSmall (..)) where
+
+import           Test.QuickCheck
+
+newtype OftenSmall a = OftenSmall a
+  deriving newtype (Show, Eq, Ord)
+
+instance (Arbitrary a, Integral a) => Arbitrary (OftenSmall a) where
```

Comment:
Maybe in the use sites of `OftenSmall` we can comment on why we use it.

Reply:
Done